docs/releasing/publishing.md

```diff
@@ -1,55 +1,62 @@
-# Steps you must complete before you release a new version of GOV.UK Frontend
+# Before you release a new version of GOV.UK Frontend
 
-1. Raise new issues in the team GitHub repositories ([govuk-frontend](https://github.com/alphagov/govuk-frontend), [govuk-frontend-docs](https://github.com/alphagov/govuk-frontend-docs), [govuk-prototype-kit](https://github.com/alphagov/govuk-prototype-kit)) to:
+1. Coordinate the final cutoff by letting the GOV.UK Design System team know we are close to releasing at stand up. Once the team agrees on final cutoff, do not add any further major changes to the release. We can still add small fixes before step 6 as long as we notify the content designer and technical writer. However, we should try to avoid adding too many fixes in this way, as it requires us to repeat some of steps 2-5.
```

Comment:
Would this work:

> Coordinate the final cutoff date by letting the GOV.UK Design System team know we are close to releasing at stand up. Once the agreed cut off date has passed, do not add any further major changes to the release.

It might help to clarify that by cutoff we mean a date rather than a process or other task.

Reply:
```suggestion
1. At stand up, tell the GOV.UK Design System team we are close to releasing so we can coordinate the final cutoff date. Once the agreed cutoff date passes, do not add any further major changes to the release. We can still add small fixes before step 7 as long as we notify the content designer and technical writer. However, we should try to avoid adding too many fixes in this way, as it requires us to repeat some of steps 4-6.
```